src/test/java/org/folio/service/finance/transaction/TransactionSummariesServiceTest.java

```diff
@@ -77,17 +87,23 @@ void testShouldTransactionsCreatedForEncumbrances() {
   }
 
   @Test
-  void testShouldCreateTransactionSummaryInStorageTransactions() {
-    // given
-
+  void testShouldCreateTransactionSummaryInStorageTransactions() throws NoSuchFieldException, IllegalAccessException {
+    // Given
     String uuid = UUID.randomUUID().toString();
-    var response = new JsonObject("{\"id\": \"" + uuid + "\"}");
-    doReturn(succeededFuture(response)).when(restClient).post(anyString(), any(), any(), any());
+    JsonObject response = new JsonObject().put("id", uuid);
+    when(restClient.post(any(RequestEntry.class), any(), any(), any(RequestContext.class)))
+      .thenReturn(succeededFuture(response));
+    OrderTransactionSummary expectedSummary = new OrderTransactionSummary().withId(uuid).withNumTransactions(2);
+    // Create an instance of your service
+    OrderTransactionSummariesService orderTransactionSummariesService2 = new OrderTransactionSummariesService(restClient);
```

Comment:
why we need to create an ew instance of the service, we should use existing orderTransactionSummariesService